File: C# Advanced/Methods-Homework/AppearanceCount/AppearanceCount.cs
// Write a method that counts how many times given number appears in a given array. Write a test program to check if the method is working correctly.

using System;
using System.Linq;

class AppearanceCount
{
    private static int CountNumberAppearance(int checkedNumber, int[] numbers)
    {
        int counter = 0;
        foreach (int number in numbers)
        {
            if (checkedNumber == number)
            {
                counter++;
            }
        }

        return counter;
    }

    static void Main()
    {
        int n = int.Parse(Console.ReadLine());

        int[] numbers = new int[n];
        numbers = Console.ReadLine().Split(' ').Select(x => int.Parse(x)).ToArray();

        int checkedNumber = int.Parse(Console.ReadLine());

        int appearanceCount = CountNumberAppearance(checkedNumber, numbers);
        Console.WriteLine(appearanceCount);
    }
}

File: JavaScript Basics/Functions/appearance-count.js
function solve(args) {
    var n = +args[0],
        numbers = args[1].split(' ').map(function(x) {
            return +x;
        }),
        chosenOne = +args[2],
        count = 0;

    for (var i = 0; i < numbers.length; i += 1) {
        if (numbers[i] === chosenOne) {
            count += 1;
        }
    }

    return count;
}
